.eslintrc.js

```diff
@@ -165,8 +165,8 @@ module.exports = {
         {
             files: [
                 'src/examples/*.{ts,tsx}',
-                'src/components/**/examples/*.{ts,tsx}',
-                'src/components/**/examples/**/*.{ts,tsx}',
+                'src/**/examples/*.{ts,tsx}',
+                'src/**/examples/**/*.{ts,tsx}',
```

Comment:
This isn't too broad, is it? @jgroth

Reply:
No it should be fine :slightly_smiling_face: